src/components/Textarea/index.tsx

```diff
@@ -96,6 +96,12 @@ export interface TextareaProps {
     onFocus?: (event: React.FocusEvent<HTMLTextAreaElement>) => void;
     /** Handler for the onBlur event */
     onBlur?: (event: React.FocusEvent<HTMLTextAreaElement>) => void;
+    /** Handler for the onKeyDown event */
+    onKeyDown?: (event: React.KeyboardEvent<HTMLTextAreaElement>) => void;
+    /** Handler for the onKeyUp event */
+    onKeyUp?: (event: React.KeyboardEvent<HTMLTextAreaElement>) => void;
+    /** Additional styling class to apply */
```

Comment:
Change the comment to `Additional styling class to the root element of the component`

Reply:
Done